### PR TITLE
{lang}[foss/2016b] R/3.3.3 w/ X11-20160819 (REVIEW)

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2010.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/4a18ffc3f421429d5402891dc3a8a3e0 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2581.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/fe2f2e285a012505e29bd208153e6162 for a full test report.

File: easybuild/easyconfigs/r/R/R-3.3.3-foss-2016b-X11-20160819.eb [hunk 1]
```diff
@@ -0,0 +1,509 @@
...
+    ('libGLU', '9.0.0'),
+    ('cairo', '1.14.6'),
+    ('libreadline', '6.3'),
+    ('ncurses', '6.0'),
+    ('bzip2', '1.0.6'),
+    ('XZ', '5.2.2'),
+    ('zlib', '1.2.8'),
+    ('SQLite', '3.13.0'),
+    ('PCRE', '8.39'),
+    ('libpng', '1.6.24'),  # for plotting in R
```
Comment (hajgato) [inline easybuild/easyconfigs/r/R/R-3.3.3-foss-2016b-X11-20160819.eb]: `cairo` 1.14.6?

Comment (hajgato) [inline easybuild/easyconfigs/r/R/R-3.3.3-foss-2016b-X11-20160819.eb]: @boegel ^

Comment (boegel) [inline easybuild/easyconfigs/r/R/R-3.3.3-foss-2016b-X11-20160819.eb]: Of course, thanks for clarifying

**Comment by boegel** [comment]
lgtm, I'll look into a version of this with `intel/2017a`

**Comment by boegel** [comment]
Going in, thanks @hajgato!

**Pull request merged**